src/spacel/provision/template/app.py

```diff
@@ -141,6 +143,22 @@ def app(self, app_region):
                 }
             }
 
+        # Custom update policy:
+        if app_region.update_policy == 'disabled':
+            del resources['Asg']['UpdatePolicy']
+        if app_region.update_policy == 'redblack':
+            resources['Asg']['UpdatePolicy'] = {
+                'AutoScalingReplacingUpdate': {
+                    'WillReplace': 'true'
+                }
+            }
+            time_pos = (resources['Lc']
```

Comment:
I do like the idea of implementing `rotate` within CloudFormation; 'tis the spacel-way: let go, let cloudformation.

I'm not sure about cache-busting 100% of the time though: if I want to add a `sources` entry (i.e. adjust a security group), I shouldn't have to wait ~300s for fresh instances (and pay 2x instance cost for an hour?).

Could we support the "redeploy without changes" case some other (more explicit) way?